features/user_can_create_account.feature

```diff
@@ -0,0 +1,29 @@
+Feature: User can create account
+    As a User,
+    In order to use the application
+    I would like to be able to create an account
+
+    Background: 
+        Given I visit the "landing" page
```

Comment:
I would remove the quotes around landing page for this step.

Reply:
Done